app/hotels/src/filter/price/PriceFilter.js

```diff
@@ -125,20 +180,32 @@ type Props = {|
   +isActive: boolean,
 |};
 
+const calculateDaysOfStay = (checkin, checkout) => {
+  if (checkin && checkout) {
+    return DateUtils.diffInDays(checkout, checkin);
+  } else {
+    return 1;
+  }
+};
+
 export default function PriceFilterWithContext(props: Props) {
   return (
     <HotelsContext.Consumer>
-      {({ currency }) => (
-        <HotelsSearchContext.Consumer>
-          {({ currentSearchStats }) => (
-            <PriceFilter
-              {...props}
-              currentSearchStats={currentSearchStats}
-              currency={currency}
-            />
-          )}
-        </HotelsSearchContext.Consumer>
-      )}
+      {({ currency, checkin, checkout }) => {
+        const daysOfStay = calculateDaysOfStay(checkin, checkout);
```

Comment:
then you could check here of daysOfStay === null and in that case just return null. If any of the dates are missing, there is obvoiusly some error from native side.